### ISSUE TITLE
Enhancement: On the domain purchase flow show which site it's being purchased for

### ISSUE THREAD

**Issue by arunsathiya** [Automattic/wp-calypso#29971]
This is more of an enhancement request. 

When users visit https://wordpress.com/domains/add, the site picker shows up and that's good.

But, often, for single-site users, when they visit https://wordpress.com/domains/add, the domain management list shows up for that particular site.

When they click `Add domain` to register the domain, they do not realize that they are purchasing a domain for the same site, and not a new site.

This flow could probably be improved, with a copy like `New domains purchased will be pointed to the same site. If you would like to create a new site instead, visit https://wordpress.com/start`, or something along those lines. I defer to design and editorial gurus for decisions on that.

#user-reports #pain-points

**Comment by alaczek** [comment]
Hi @arunsathiya! I'm not sure I understand the issue here. 

> When they click Add domain to register the domain, they do not realize that they are purchasing a domain for the same site, and not a new site.

Since in this scenario customer has only one site, I'm not sure why this is a bad thing? Isn't it expected to purchase a domain for an existing site? 

What is the flow that gets them to https://wordpress.com/domains/add? Are you assuming here that customer would come to domains page thinking this is a way to start a new site? Is this something that you encountered?

**Comment by arunsathiya** [comment]
>Since in this scenario customer has only one site, I'm not sure why this is a bad thing? Isn't it expected to purchase a domain for an existing site?

I think I might not have been very clear earlier, but let me try to rephrase.

- I am now a WordPress.com user with one site
- I visit https://wordpress.com/domains/add/
- The current site's domain is automatically appended to it, and the link becomes https://wordpress.com/domains/add/example.wordpress.com 
- At this stage, a search box is also seen where I enter a new domain name, hoping to create a new website
- But, only after checkout, I realise it just adds the domain to the existing site and that I am not creating a new website with that domain

This is mostly a non-issue and I am probably over-imagining the flow. :) Would this issue be something where we need to add `Needs design review` label? 🤔 

If we feel that this is a non-issue, let's totally close it! 

>What is the flow that gets them to https://wordpress.com/domains/add?

Very often, HEs share this link on chat/tickets, when supporting users, and that "Site picker" (which appears when having multiple sites) is something I find helpful.

**Comment by alaczek** [comment]
> At this stage, a search box is also seen where I enter a new domain name, hoping to create a new website

I guess this is where I get hung up - why would you/customer think you're creating a new site at this point? Maybe this is a bigger issue of customers not knowing which site they are editing?

> Very often, HEs share this link on chat/tickets, when supporting users, and that "Site picker" (which appears when having multiple sites) is something I find helpful.
Trying to understand - would this still be helpful when customer has just one site? I examined the site picker, and as far as I can tell it's not possible to create a new site from there AND purchase a domain for it (even if you input a new site name that does not exist yet). So I'm not seeing what would be accomplished by sending there a customer that has only one site.

> This is mostly a non-issue and I am probably over-imagining the flow. :)

No worries, it's always good to discuss and explore ideas! :) @sixhours maybe you have thoughts on this?

**Comment by sixhours** [comment]
I'd want to do some research into this before making changes -- do we have Happiness feedback that suggests this is a common source of confusion? Can you share a few specific reports or user tests?

If so, let's definitely do some design thinking around it. If not, I think we can close it.

**Comment by arunsathiya** [comment]
I have run into a couple of interactions with users where confusions have raised, but there is no feedback tracker from Happiness that I am aware of. This might indeed be something that isn't largely a concern. I am closing this report for now. We can always reopen as needs be. 👍🏽

**Comment by arunsathiya** [comment]
I nudged for a discussion here. p7DVsv-6Vg-p2

**Comment by sixhours** [comment]
Reopening this since it looks like there's precedent for an improvement here based on Happiness feedback. Thanks for spearheading that, Arun!

**Comment by arunsathiya** [comment]
I came across this issue by @lakellie that can be a potential solution:

https://github.com/Automattic/wp-calypso/issues/27932

There is a open PR to implement this as well, from an outside contributor, and it's being reviewed by @aidvu and @ramonjd:

https://github.com/Automattic/wp-calypso/pull/34990

Closing this in favor for the other - https://github.com/Automattic/wp-calypso/issues/27932
